lib/segment/src/index/field_index/full_text_index/tokenizers.rs

```diff
@@ -172,6 +188,21 @@ mod tests {
         assert_eq!(tokens.get(1), Some(&"мир".to_owned()));
     }
 
+    #[test]
+    fn test_multilingual_tokenizer() {
+        let text = "本日の日付は";
+        let mut tokens = Vec::new();
+        MultilingualTokenizer::tokenize(text, |token| tokens.push(token.to_owned()));
+        eprintln!("tokens = {tokens:#?}");
+        assert_eq!(tokens.len(), 6);
+        assert_eq!(tokens.get(0), Some(&"本".to_owned()));
+        assert_eq!(tokens.get(1), Some(&"日".to_owned()));
+        assert_eq!(tokens.get(2), Some(&"の".to_owned()));
+        assert_eq!(tokens.get(3), Some(&"日".to_owned()));
+        assert_eq!(tokens.get(4), Some(&"付".to_owned()));
+        assert_eq!(tokens.get(5), Some(&"は".to_owned()));
+    }
```

Comment:
Hi, I just noticed this test case and there might be something wrong to me.
It is clearly Japanese text, but I don't think it is correct tokenization.
`本日の日付は` should be splitted into four tokens `本日` / `の` / `日付` / `は`. 

I am a contributor of https://github.com/lindera-morphology/lindera (the Japanese tokenizer internally used by `charabia` tokenizer), but I'm not familiar with `charabia`.

@mosuka, do you know anything about the behavior of charabia?

Reply:
Ok, I missed the previous comment https://github.com/qdrant/qdrant/pull/2260#issuecomment-1636337045 so never mind.

I'd like to have clear documentation for the change; while it may be acceptable for general "multilingual" support, it would not be valid "CJK support" without the full Charabia features (language-specific dictionaries).

Reply:
we will document it for those who want to have full support

Reply:
Thank you, I fully understand the design choice here not to bloat the Qdrant itself. Just wanted to clarify what we will have with this change.

Reply:
@mocobeta @generall 
Charabia performs language detection from the input string and then splits it into words using a tokenizer prepared for the detected language.
UniDic is used by default.

https://github.com/meilisearch/charabia/blob/5f8abfe561bd985f7288a9a196b7e3ebeb773dc4/charabia/src/segmenter/japanese.rs#L25

Reply:
`charabia = { version = "0.7.2", default-features = false }`

The above is specified in Cargo.toml, but I believe that would not allow all the language-specific tokenizers available in Charabia.